airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -29,6 +30,20 @@
 )
 
 
+@pytest.mark.parametrize(
+    "login_status_code, login_json_resp, expected_error_msg",
+    [(400, {"error": "invalid_grant", "error_description": "expired access/refresh token"}, AUTHENTICATION_ERROR_MESSAGE_MAPPING.get("expired access/refresh token"))]
```

Comment:
can you add two test cases?
1. 400 error with a different error message -> we shouldn't raise a config error
2. different error code -> we shouldn't raise a config error

Reply:
added